packages/uicore/src/components/ModalDialog/ModalDialog.tsx

```diff
@@ -199,12 +198,11 @@ export const ModalDialog: FC<ModalDialogProps> = ({
       className={cx(className, css.container, ...modifiers)}
       style={style}
       {...dialogProps}>
-      {showOverlay ? (
-        <OverlaySpinner className={css.overlayContainer} show={true}>
-          {modalContent}
-        </OverlaySpinner>
-      ) : (
-        modalContent
+      {modalContent}
```

Comment:
This will always show modal content even if only spinner needs to shown, right?

Reply:
Yes @chetanvekariya-harness. That is the requirement. To show the model content behind the overlay.